index.md

```diff
@@ -3,72 +3,134 @@ title: Amazon Ion
 
-### Latest News
+In contrast, Ion's rich type system enables unambiguous semantics for data (e.g., a timestamp value can be encoded using the timestamp type).  The following illustrates some of the features of the Ion type system:
 
----
-{% for post in site.posts limit:2 %}
-  **<a href="{{site.baseurl}}{{post.url}}">{{ post.title }}</a>**<br/>
-  *{{post.date | date_to_long_string}}*<br/>
-  {{post.content}}
-{% endfor %}
----
-Visit the [News][7] page for more announcements about Amazon Ion.
+* **timestamp:**  arbitrary precision date / timestamps
+```
+2003-12-01T
+2010-03-22T18:00:00Z
+2019-05-01T18:12:53.472-0800
+```
+
+* **int:**  arbitrary size integers
+```
+0
+-1
+12345678901234567890...
+```
+
+* **decimal:**  arbitrary precision, base-10 encoded real numbers
+```
+0.
+-1.2
+3.141592653589793238...
+6.62607015d-34
+```
+
+* **float:**  32-/64-bit IEEE-754 floating-point values
+```
+0e0
+-1.2e0
+6.02e23
+-inf
+```
+
+* **symbol:**  provides efficient encoding for frequently occurring strings
+```
+inches
+dollars
+'high-priority'    // symbols with special characters are enclosed in single-quotes
+```
 
-<br/>
+* **blob:**  binary data
+```
+{{"{{"}} aGVsbG8= {{}}}}
+```
 
-### Example {#example}
+* **annotation:**  metadata associated with a value
+```
+dollars::100.0
+height::inches::72
+lotto_numbers::[7, 9, 19, 40, 42, 44]
+```
+
+The [Specification][10] provides an overview of the full set of Ion types.
+
+### Binary Encoding
+
+Ion provides two encodings for the type system:  human-readable text (as shown above), and a space- and read-efficient binary encoding.  When binary-encoded, every Ion value is prefixed with the value's type and length.  The following illustrates a few of the efficiences provied by Ion's binary encoding:
+
+* The following timestamp encoded as a JSON string requires 26 bytes:  "2017-07-26T16:30:04.076Z".  This timestamp requires just 11 bytes when encoded in Ion binary:
+```
+6a 80 0f e1 87 9a 90 9e 84 c3 4c
+```
+That first byte `6a` indicates the value is a timestamp (type 6) represented by the subsequent 10 bytes (that's what the `a` represents).  If this particular timestamp value is not of interest, a reader can simply skip 10 bytes.  While this may seem trivial, the efficiencies can be substantial if some or all of the bytes for large container values can be skipped!
+
+* Binary encoding of a symbol replaces the text of a symbol with an integer that can be resolved to the original text via a symbol table.  This can result in substantial space savings for symbols that occur frequently!
+
+* While blob data is base-64 encoded in text (which produces 4 bytes for every 3 bytes of the original data), a blob encoded as Ion binary is simply encoded as is&mdash;no base-64 expansion required!
+
+Similar space efficiencies are found in other aspects of Ion's binary encoding, but rather than being exhaustive here, perhaps you'd like to . . .
```

Comment:
The sentence here lets me to infer that I can use the JS area below to try more examples and see space savings. I would split this sentence into 2 

1. Similar space efficiencies are found in other aspects of Ion's binary encoding. 
2. Give Ion a try!

Make sentence 1, the last sentence of the preceding paragraph. 

Make sentence 2. a new paragraph (add empty line between the preceding paragraph) immediately followed by the JS area.

Reply:
change 1. to be a bullet, made 2. the header of the subsequent section